src/main/steps/respondent/upload-document/need-permission/content.ts

```diff
@@ -0,0 +1,57 @@
+import { TranslationFn } from '../../../../app/controller/GetController';
```

Comment:
folder name shd be submit-extra-evidence

Reply:
done this.